Jenkinsfile

```diff
@@ -17,6 +17,22 @@ String getBuildLog() {
     return currentBuild.rawBuild.getLogFile().getText()
 }
 
+boolean areIgnoredFiles(HashSet<String> changedFiles) {
+  HashSet<String> ignoredFiles = [
+    "CHANGELOG.md",
+    "README.md"
```

Comment:
Add a comment that this is the full path to the files, relative to the root of the repo?

Reply:
Handled with commit e3ecd5fe538df3862b54e194b822fdd4f27cd987